WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -94,6 +95,14 @@ protected SendControlViewModel(Wallet wallet, string title)
 			SetFeeTargetLimits();
 			FeeTarget = Global.UiConfig.FeeTarget;
 			FeeDisplayFormat = (FeeDisplayFormat)(Enum.ToObject(typeof(FeeDisplayFormat), Global.UiConfig.FeeDisplayFormat) ?? FeeDisplayFormat.SatoshiPerByte);
+			_transactionFeeInfo = this.WhenAnyValue(
+				x => x.UsdFee,
+				x => x.UsdExchangeRate,
+				x => x.FeePercentage,
+				x => x.FeeRate,
+				x => x.EstimatedBtcFee,
+				(a, b, c, d, e) => new TransactionFeeInfo(a, b, c, d, e))
+				.ToProperty(this, x => x.TransactionFeeInfo, scheduler: RxApp.MainThreadScheduler);
```

Comment:
This part generates a warning: 

```
Warning	CS8619	Nullability of reference types in value of type 'ObservableAsPropertyHelper<TransactionFeeInfo?>' doesn't match target type 'ObservableAsPropertyHelper<TransactionFeeInfo>'.	WalletWasabi.Gui	C:\work\csharp\Wasabi\WalletWasabi\WalletWasabi.Gui\Controls\WalletExplorer\SendControlViewModel.cs	98	Active
```

![image](https://user-images.githubusercontent.com/58662979/90751318-4400c000-e2d6-11ea-9de6-b0850bae3549.png)

Reply:
I think it's because `x.UsdFee` and others are not assigned but they are still denoted as non-nullable.

Reply:
Not sure what the best way to fix this is, do you how?

Also, not sure this is the highest priority issue as on my machine there are >600 warnings on `master`

Reply:
Nvm fixed